sw/Modules/Comms/Inc/can_sensors.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * can_sensors.h
+ *
+ *  Created on: Mar 19, 2025
+ *      Author: Hardy Yu
+ */
+#ifndef CAN_SENSORS_H_
+#define CAN_SENSORS_H_
+
+#include <stdint.h>
+#include "canard.h"
+#include "dronecan_msgs.h"
+#include "imu.h"
+
+typedef enum {
+    COMMS_STATUS_OK = 0,
+    COMMS_STATUS_ERR = 1,
+} CAN_COMMS_STATUS_E;
+
+void
+convertImuToDroneCAN (const ImuData_S *src,
+                    // const uint8_t imu_id,
```

Comment:
Would be "correct" to remove comment or add explanation for why it's commented out.